gengo/gengo.py

```diff
@@ -399,6 +417,22 @@ def signAndRequestAPILatest(self, fn, base, query_params, post_data={},
                               # SSL here ...
                               verify=False)
 
+    def modifyURLAttachments(self, obj):
+        """
+        modifyURLAttachments handles url attachments for a given object
```

Comment:
describe what purpose it modifies for.
reflect its responsibility to the method name.